### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by AJenbo**

### CODE REVIEW

**Comment by julealgon** [comment]
What's up with the build failures?

**Comment by AJenbo** [comment]
> What's up with the build failures?

Turns out  			`sgnMusicTrack = (_music_id)nTrack; ` is needed

**Comment by AJenbo** [comment]
Really struggeling with this new fancy emoji compatible git client...

**Comment by AJenbo** [comment]
@julealgon should be good now

File: Source/quests.h [hunk 1]
```diff
@@ -60,7 +55,7 @@ struct QuestData {
 	Uint8 _qdtype;
 	Uint8 _qdrnd;
 	Uint8 _qslvl;
-	quest_gametype _qflags;
+	bool isSinglePlayerOnly;
```
Comment (julealgon) [inline Source/quests.h:58]: Another huge readability booster.

**Review by julealgon — APPROVED** [review]

**Pull request merged**